### ISSUE TITLE
High increasing load on the CPU

### ISSUE THREAD

**Issue by dotX12** [Krukov/cashews#93]
Found on the master branch.

after updating our product in production to the master branch (cashews), the CPU load increased. 

![image](https://user-images.githubusercontent.com/64792903/190115387-0538825b-f78b-4e73-b59b-b3d0a45937f9.png)

indicators earlier.

![image](https://user-images.githubusercontent.com/64792903/190115439-d3003b46-32f7-4b04-bbce-db9dbdea0bdd.png)

i use:

```
cashews = {git = "https://github.com/Krukov/cashews.git", rev = "master"}
```

```python3
cache = Cache()
cache.setup(
    settings_redis.dsn,
    client_side=True,
    retry_on_timeout=True,
    hash_key=settings_redis.hash,
    pickle_type="sqlalchemy",
)
```

**Comment by dotX12** [comment]
UPD:


![image](https://user-images.githubusercontent.com/64792903/190115575-2684ddcf-ccd0-4b73-a4f1-ab99ebb38dee.png)
It's too early to say, the chart needs to stabilize.
In 6 hours I will send a new stats.

On 4.6.0 this problem does not exist.

**Comment by dotX12** [comment]
I guess it has to do with: https://github.com/Krukov/cashews/issues/83

**Comment by Krukov** [comment]
@dotX12 
Thanks for reporting that CPU issue, Il try to investigate it.
Also `client_side=True` can be a reason.  Did you discover any issues with redis or cache itself at that time?

**Comment by Krukov** [comment]
And did it happen right after deploy a new version with cashes on master or after while?

**Comment by dotX12** [comment]
> @dotX12 Спасибо, что сообщили о проблеме с процессором, я попытаюсь ее исследовать. Тоже `client_side=True`может быть причиной. Обнаружили ли вы какие-либо проблемы с Redis или самим кешем в то время?

The redis statistics show nothing interesting.
https://telegra.ph//file/0a101534b3a0fcf2209c3.png

**Comment by dotX12** [comment]
> And did it happen right after deploy a new version with cashes on master or after while?

within a few hours there was an increase in load, after which the system failed.
https://telegra.ph//file/d45753a47fa83503c7e6d.png

**Comment by Krukov** [comment]
Sorry for delay. I found an error. It was not related to serializer, it was related to new check for initialized connection that was not released yet. 

Fixed in 4.7.1
